index.html

```diff
@@ -1,95 +1,34 @@
-                <li><hr class="dropdown-divider"></li>
-                <li><a class="dropdown-item" href="#">Something else here</a></li>
-              </ul>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link disabled">Disabled</a>
-            </li>
-          </ul>
-          <form class="d-flex">
-            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-            <button class="btn btn-outline-success" type="submit">Search</button>
-          </form>
-        </div>
-      </div>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <title>Real food made Real Good!</title>
+
+</head>
+<!-- Header Section -->
+<header>
+    Real food made Real Good!
+</header>
+<!-- Main section -->
+<div class="container">
+    <div class="center_image"><img src="recipes/images/bon.png" alt="birds"></div>
+    <h1> Real food made Real Good!</h1>
+    <!--Bottom Navigation section -->
+    <nav>
+        <ul>
+            <li>Spinach and Strawberry Salad</li>
+            <li><a href="#"> Gordon's Berry Breakfast Drink</a></li>
+            <li><a href="#"> Cream Cheese Frosting II </a></li>
+        </ul>
     </nav>
-    <!--Story one and two-->
-    <div class="navbar navbar-expand-lg navbar-light bg-light">
-      <div class="card-body">
-        <section id="welcome-section"><h1>Welcome To My Personal Page</h1></section>
-    <section id="projects"><a href="https://github.com/Loltolo-Lesapiti
-      " class="link-primary" id="profile-link">This is my Project section </a>
-      <p class ="project-tile"> This is my project title section. This are the projects I have done so far
-        <a href="https://www.microverse.org
-        " class="link-primary">Here</a>
-      </p>
-      
-    </section>
-      </div>
-    </div>
-    
-    
-
-    
-  
-    <!-- Optional JavaScript; choose one of the two! -->
-
-    <!-- Option 1: Bootstrap Bundle with Popper -->
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
-
-    <!-- Option 2: Separate Popper and Bootstrap JS -->
-    <!--
-    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js" integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js" integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13" crossorigin="anonymous"></script>
-    -->
-  </body>
-</html>
-
-
-
-
-
-
-
+</div>
+<div>
+    my name is Alfred Kinyua and i am a software developer 
+</div>
```

Comment:
Please consider using the <P> Html element to your content. Also consider appropriate <section> tags in your web page.